moto/ec2/responses/transit_gateway_route_tables.py

```diff
@@ -29,6 +29,42 @@ def delete_transit_gateway_route_table(self):
         template = self.response_template(DELETE_TRANSIT_GATEWAY_ROUTE_TABLE_RESPONSE)
         return template.render(transit_gateway_route_table=transit_gateway_route_table)
 
+    def create_transit_gateway_route(self):
+        transit_gateway_attachment_id = self._get_param("TransitGatewayAttachmentId")
+        destination_cidr_block = self._get_param("DestinationCidrBlock")
+        transit_gateway_route_table_id = self._get_param("TransitGatewayRouteTableId")
+        blackhole = True if self._get_param("Blackhole") else False
```

Comment:
Could the result of `self._get_param("Blackhole")` also contain the string `"false"`?

Perhaps better change to something like:
```
TRUE_VALUES = [True, "true", "True"]
...
         blackhole = self._get_param("Blackhole") in TRUE_VALUES
```

Reply:
We can have one util for which will convert str(True or False) to boolean type.